include/oneapi/dpl/internal/serial_numeric_impl.h

```diff
@@ -17,7 +17,9 @@
 #ifndef _ONEDPL_SERIAL_NUMERIC_IMPL_H
 #define _ONEDPL_SERIAL_NUMERIC_IMPL_H
 
-#if (__cplusplus >= 201703L)
+#include "common_config.h"
```

Comment:
The same comment as in file `../internal/serial_algorithm_impl.h`

Reply:
Discussed in https://github.com/oneapi-src/oneDPL/pull/678#discussion_r1066725656